pyoos/parsers/nerrs.py

```diff
@@ -16,25 +16,26 @@ def units(param):
     # http://cdmo.baruch.sc.edu/documents/manual.pdf
     units = {
         # Water Quality
-        'Temp'      :       '°C',
```

Comment:
True :stuck_out_tongue_winking_eye:

(I just don't want to forget to test **this** part of the code explicitly once we get the chance.)